tests/benchmarks/test_parquet.py

```diff
@@ -24,6 +24,7 @@ def parquet_cluster(dask_env_variables):
         package_sync=True,
         environ=dask_env_variables,
         backend_options={"send_prometheus_metrics": True},
+        tags=gitlab_cluster_tags,
```

Comment:
Should we be changing instance types here too?

Reply:
Nope, I'm just changing places that are currently using t3.